api/src/uvisor-input.S

```diff
@@ -106,6 +106,9 @@ uvisor_config:
     /* Privileged system IRQ hooks */
     .long __uvisor_priv_sys_irq_hooks
 
+    /* uVisor Lib Functions for use by uVisor in unprivileged mode */
```

Comment:
This comment is a bit misleading, shouldn't it say something like "Default box initialization handler"? It's the system-wide function that is run by every box at init time.